<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAddDorama</class>
 <widget class="QDialog" name="DialogAddDorama">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>871</width>
    <height>597</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add dorama</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedKingdom"/>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="HLay_0">
     <item>
      <layout class="QVBoxLayout" name="VLay_0">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <layout class="QHBoxLayout" name="HLay_1">
         <item>
          <widget class="QLabel" name="label">
           <property name="font">
            <font>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Dorama</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="HSpacer_0">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QCheckBox" name="CheckBox_LookLater">
           <property name="text">
            <string>Want to look</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="CheckBox_Editing">
           <property name="text">
            <string>Editing</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLineEdit" name="LineEdit_Title">
         <property name="maxLength">
          <number>128</number>
         </property>
         <property name="placeholderText">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="VLay_AltTitle">
         <property name="topMargin">
          <number>0</number>
         </property>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QCheckBox" name="CBox_Year">
           <property name="text">
            <string>Year:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="SpinBox_Year">
           <property name="minimum">
            <number>1900</number>
           </property>
           <property name="maximum">
            <number>9999</number>
           </property>
           <property name="value">
            <number>2000</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="HSpacer_1">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QLabel" name="Lbl_Season">
           <property name="text">
            <string>Season:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="SpinBox_Season"/>
         </item>
         <item>
          <spacer name="HSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTabWidget" name="TabWidget_Series">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>120</height>
          </size>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_allSeries">
          <attribute name="title">
           <string>Total series</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout">
           <item row="0" column="1">
            <widget class="QLabel" name="Lbl_Spec1">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>Special</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QSpinBox" name="SpinBox_aTV">
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="Lbl_Film1">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>Movie</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="Lbl_TV1">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>40</height>
              </size>
             </property>
             <property name="text">
              <string>TV</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSpinBox" name="SpinBox_aMovie">
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="SpinBox_aSpec">
             <property name="maximum">
              <number>9999</number>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <spacer name="VlSpacer">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_lookSeries">
          <attribute name="title">
           <string>Sees</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="Lbl_TV2">
             <property name="text">
              <string>TV</string>
             </property>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QSpinBox" name="SpinBox_vMovie">
             <property name="maximum">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QSpinBox" name="SpinBox_vSpec">
             <property name="maximum">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QSpinBox" name="SpinBox_vTV">
             <property name="maximum">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="Lbl_Spec2">
             <property name="text">
              <string>Special</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="Lbl_Film2">
             <property name="text">
              <string>Movie</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <spacer name="VSpacer_2">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>40</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QTabWidget" name="TabWidget_Info">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab">
          <attribute name="title">
           <string>Genres</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLineEdit" name="LineEdit_Tags">
             <property name="text">
              <string/>
             </property>
             <property name="maxLength">
              <number>256</number>
             </property>
             <property name="placeholderText">
              <string>Enter through a comma, or choose from the list lower</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QListView" name="ListView_Tags">
             <property name="frameShape">
              <enum>QFrame::Panel</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustToContents</enum>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::NoEditTriggers</set>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::MultiSelection</enum>
             </property>
             <property name="isWrapping" stdset="0">
              <bool>true</bool>
             </property>
             <property name="resizeMode">
              <enum>QListView::Adjust</enum>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_2">
          <attribute name="title">
           <string>Description</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <widget class="QPlainTextEdit" name="PlainTextEdit_Description">
             <property name="placeholderText">
              <string>Description</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_roles">
          <attribute name="title">
           <string>In roles</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QPlainTextEdit" name="PlainTextEdit_Actors">
             <property name="placeholderText">
              <string>In roles</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_3">
          <attribute name="title">
           <string>Media</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="HLayout_Dir">
             <item>
              <widget class="QLineEdit" name="LineEdit_Dir">
               <property name="maxLength">
                <number>256</number>
               </property>
               <property name="placeholderText">
                <string>File</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QToolButton" name="toolButton">
               <property name="toolTip">
                <string/>
               </property>
               <property name="text">
                <string>*</string>
               </property>
               <property name="icon">
                <iconset resource="resource.qrc">
                 <normaloff>:/images/folder.png</normaloff>:/images/folder.png</iconset>
               </property>
               <property name="iconSize">
                <size>
                 <width>24</width>
                 <height>24</height>
                </size>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLineEdit" name="LineEdit_URL">
             <property name="maxLength">
              <number>256</number>
             </property>
             <property name="placeholderText">
              <string>URL</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="VSpacer_0">
             <property name="orientation">
              <enum>Qt::Vertical</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>20</width>
               <height>138</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="ImageCover" name="Lbl_ImageCover">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>3</horstretch>
         <verstretch>1</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>180</width>
         <height>540</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>275</width>
         <height>825</height>
        </size>
       </property>
       <property name="baseSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Drop image here, or a double click for a file choice</string>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="resource.qrc">:/images/DropHere.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="textInteractionFlags">
        <set>Qt::NoTextInteraction</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="BtnBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ImageCover</class>
   <extends>QLabel</extends>
   <header location="global">imagecover.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
